bool check
---
cobot_pump_ros/waypoint[] waypoints